pkg/cli/run.go

```diff
@@ -314,6 +315,13 @@ func (s *Run) update(ctx context.Context, c client.Client, imageSource imagesour
 	updateOpts.Replace = s.Replace
 	updateOpts.Image = image
 	updateOpts.DeployArgs = deployArgs
+	if s.Refresh {
+		refreshAnnotation, err := v1.ParseScopedLabels("acorn.io/refresh=true")
+		if err != nil {
+			return nil, false, err
+		}
+		updateOpts.Annotations = append(updateOpts.Annotations, refreshAnnotation...)
```

Comment:
I would like to avoid having to add this annotation to the app spec. Can we do something here to only put the annotation on the `AppInstance` object?

The reason being that this actually changes the app (which bumps the generation).